tailwind.config.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
       colors: {
         'wri-black': '#191716',
         'wri-blue': '#05377b',
+        'wri-purple': '#6457A5',
```

Comment:
My only comment is to maybe move it down a couple rows to try to keep this list alphabetical. That is NOT a blocker, though. Feel free to merge it either way.